app/controllers/home_controller.rb

```diff
@@ -99,5 +99,14 @@ def index
 
     @filters = filters.to_json
     @projects = Project.all.order(start_date: :desc).to_json
+
+    respond_to do |format|
+      format.html
+      format.csv { send_data Project.to_csv(params[:ids]),
```

Comment:
This is just for testing purposes right?

Reply:
Ah I thought we needed this `format.csv` respond_to for it to work? Is there another way? I named the variable `ids` maybe I could use a better name?

Reply:
the params and the concepts are fine, but I think this should go in another action as the `index` one will be mainly for display purposes rather then download.

Reply:
Thanks @Levia would an acceptable name for the action be something like `to_csv` or `download_csv` for example?

Reply:
Or simply `download`?

Reply:
I think `download` should be fine